backend/src/models/User.js

```diff
@@ -0,0 +1,54 @@
+import uuid from 'uuid/v4'
+
+module.exports = {
+  id: { type: 'string', primary: true, default: uuid }, // TODO: should be type: 'uuid' but simplified for our tests
+  actorId: { type: 'string', allow: [null] },
+  name: { type: 'string', min: 3 },
+  email: { type: 'string', lowercase: true, email: true },
+  slug: 'string',
+  encryptedPassword: 'string',
+  avatar: { type: 'string', allow: [null] },
+  coverImg: { type: 'string', allow: [null] },
+  deleted: { type: 'boolean', default: false },
+  disabled: { type: 'boolean', default: false },
+  role: { type: 'string', default: 'user' },
```

Comment:
so, basically you changed this to have a default of `user`
the other changes to this `PR` are just refactoring to extract out the models into their own files, correct?